onnxscript/tests/function_libs/torch_lib/ops_test.py

```diff
@@ -32,9 +32,13 @@
 from onnxscript._internal import version_utils
 from onnxscript.tests.function_libs.torch_lib import ops_test_common, ops_test_data
 
-# Test only float32 inputs. All dtypes will be tested on the generated symbolic functions.
+# All dtypes will be tested on the generated symbolic functions.
```

Comment:
How is float32 and float16 all dtypes?

Reply:
INT64, INT32, INT8, etc. will be gradually add in, so it is for all dtypes.